src/Arcus.Observability.Tests.Unit/Serilog/ApplicationInsightsTelemetryConverterTests.cs

```diff
@@ -0,0 +1,353 @@
+            Assert.NotNull(logEvent);
+
+            var converter = ApplicationInsightsTelemetryConverter.Create();
+
+            // Act
+            IEnumerable<ITelemetry> telemetries = converter.Convert(logEvent, formatProvider: null);
+
+            // Assert
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.RequestMethod);
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.RequestHost);
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.RequestDuration);
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.ResponseStatusCode);
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.RequestTime);
+            AssertDoesNotContainLogProperty(logEvent, RequestTracking.ResponseStatusCode);
+            Assert.Collection(telemetries, telemetry =>
+            {
+                var requestTelemetry = Assert.IsType<RequestTelemetry>(telemetry);
+                Assert.Equal("GET /api/v1/health", requestTelemetry.Name);
+                Assert.Equal(TruncateToSeconds(DateTimeOffset.UtcNow), requestTelemetry.Timestamp);
+                Assert.Equal(duration, requestTelemetry.Duration);
+                Assert.Equal("200", requestTelemetry.ResponseCode);
+                Assert.True(requestTelemetry.Success);
+                Assert.Equal(operationId, requestTelemetry.Id);
+                Assert.Equal(new Uri("https://localhost/api/v1/health"), requestTelemetry.Url);
+
+                AssertContainsTelemetryProperty(requestTelemetry, "Client", "https://localhost");
+                AssertContainsTelemetryProperty(requestTelemetry, "ContentType", "application/json");
+            });
+        }
+
+        private static HttpRequest CreateStubRequest(HttpMethod httpMethod, string requestScheme, string host, string path)
+        {
+            var request = new Mock<HttpRequest>();
+            request.Setup(req => req.Method).Returns(httpMethod.ToString().ToUpper);
+            request.Setup(req => req.Scheme).Returns(requestScheme);
+            request.Setup(req => req.Host).Returns(new HostString(host));
+            request.Setup(req => req.Path).Returns(path);
+
+            return request.Object;
+        }
+
+        private static HttpResponse CreateStubResponse(HttpStatusCode statusCode)
+        {
+            var response = new Mock<HttpResponse>();
+            response.Setup(res => res.StatusCode).Returns((int) statusCode);
+
+            return response.Object;
+        }
+
+        private static DateTimeOffset TruncateToSeconds(DateTimeOffset dateTimeOffset)
+        {
+            return new DateTimeOffset(
+                dateTimeOffset.Year, dateTimeOffset.Month, dateTimeOffset.Day, dateTimeOffset.Hour, dateTimeOffset.Minute, dateTimeOffset.Second, dateTimeOffset.Offset);
+        }
+
+        [Fact]
+        public void LogHttpDependency_WithHttpDependencyMessagePrefix_CreatesHttpDependencyTelemetry()
```

Comment:
These `WithXXXMessagePrefix` are not really correct as we don't care about them here, no?

Maybe just use `Valid` or `ValidInput` or so?

Reply:
Removed the `MessagePrefix` part.

Reply:
I don't see it but presume you still have to push it then?

Reply:
Checking latest version helps 🤦‍♂